src/transaction.py

```diff
@@ -108,7 +108,10 @@ class StakingInterest(Transaction):
 
 
 class Airdrop(Transaction):
-    pass
+    taxable = True
```

Comment:
Default value needs to be defined in __init__. If you define it here. It's a general class variable. Alle classes share this variable